analyzers/its/expected/Nancy/Nancy.ViewEngines.Razor--net452-S5693.json

```diff
@@ -0,0 +1,30 @@
+{
+"issues":  [
+{
+"id":  "S5693",
+"message":  "Make sure the content length limit is safe here.",
+"location":  {
+"uri":  "sources\Nancy\samples\Nancy.Demo.Hosting.Owin\Web.config",
```

Comment:
I see all issues which are raised are raised on the same file: `sources\Nancy\samples\Nancy.Demo.Hosting.Owin\Web.config` , even if the projects that get analyzed are different

Could you look why this is happening? Does this mean that in SonarQube multiple issues for the same locations will be raised?

Reply:
The way we build FilesToAnalyze.txt in our ITs - they are all of the same size and contain all the files for this project. That's why you see this entry even in unrelated projects.

Reply:
hm ok, we should add a comment somewhere about this...
Not very sure where - in the Target that adds the FilesToAnalyze.txt

but that would be a separate PR

Reply:
(I am leaving this thread open for a reader of the PR to easily find this conversation in case he/she will wonder why this is happening)